source/GameData.cpp

```diff
@@ -1061,7 +1069,15 @@ void GameData::LoadFile(const string &path, bool debugMode)
 		else if(key == "mission" && node.Size() >= 2)
 			missions.Get(node.Token(1))->Load(node);
 		else if(key == "outfit" && node.Size() >= 2)
-			outfits.Get(node.Token(1))->Load(node);
+		{
+			// Allow outfits to inherit attributes from parents
+			const string &name = node.Token((node.Size() > 2) ? 2 : 1);
+			outfits.Get(name)->Load(node);
+			
+			// If this outfit is a variant add it to the map of variants
+			if(node.Size() > 2)
+				outfitVariants[outfits.Get(node.Token(1))].emplace_back(outfits.Get(name));
```

Comment:
I'm not too keen on this method of making `require` work, as it's not as if we keep track of all ship variants in the same way. Not certain on how else it could be done at the moment, though.